tests/src/python/test_qgslayertreeview.py

```diff
@@ -90,6 +90,16 @@ def testDefaultActions(self):
         show_in_overview.trigger()
         self.assertEqual(view.currentNode().customProperty('overview', 0), False)
 
+    def testMoveToTopAction(self):
+        """Test move to top action"""
+        view = QgsLayerTreeView()
+        view.setModel(self.model)
+        actions = QgsLayerTreeViewDefaultActions(view)
+        self.assertEqual(self.project.layerTreeRoot().layerOrder(), [self.layer, self.layer2, self.layer3])
+        view.setCurrentLayer(self.layer3)
+        movetotop = actions.actionMoveToTop()
+        movetotop.trigger()
+        self.assertEqual(self.project.layerTreeRoot().layerOrder(), [self.layer3, self.layer, self.layer2])
 
```

Comment:
fix indentation (run ```scripts/prepare-commit.sh```)

Reply:
I ran prepare-commit before my last commit and it said "nothing was modified". I think the indentation is the same as the test above. Can you be more specific on what needs to be changed?

Reply:
it is necessary an empty line and is what travis failure (code_layout build) was pointing:
```
--- tests/src/python/test_qgslayertreeview.py.prepare	2018-04-03 01:01:09.673477272 +0600
+++ tests/src/python/test_qgslayertreeview.py	2018-04-03 01:01:09.949480992 +0600
@@ -101,5 +101,6 @@
         movetotop.trigger()
         self.assertEqual(self.project.layerTreeRoot().layerOrder(), [self.layer3, self.layer, self.layer2])
 
+
 if __name__ == '__main__':
     unittest.main()
```

Reply:
Thanks! New line added and I have pushed it to the old commit.